android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.2'
-  compile "com.google.android.gms:play-services-base:9.2.0"
-  compile 'com.google.android.gms:play-services-maps:9.2.0'
+  compile 'com.facebook.react:react-native:+'
+  compile "com.google.android.gms:play-services-base:8.4.0"
+  compile 'com.google.android.gms:play-services-maps:8.4.0'
```

Comment:
In my Android sim (Android 6 (23) - avd + Google APIs on Nexus 5X) maps won't show.  `logcat` said that Play Services was out of date.  This will require updates on devices.

Also, (just fyi, not really an issue), it does require us to update our dev Google Play services to version 31 (in Android SDK Manager), but that's not a huge deal, re-running that as a dev isn't a big problem and we can easily put that in a readme file.

I figure, once we start using 9.x features, it's time to set the dependency?  Thinking outloud.